controllers/webhook/runcompletion_event_trigger_unit_test.go

```diff
@@ -117,6 +118,22 @@ var _ = Context("EventDataToPbRunCompletion", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("returns an empty slice for no artifacts when event data is converted to proto runcompletion event", func() {
```

Comment:
```suggestion
		It("returns an empty slices when there are no artifacts", func() {
```

don't need the `when event data is converted to proto runcompletion event` as that's already in the `When` for the test

Reply:
done in 776746f912d61af47d69f5eec9c65c0366752a5e